script.screensaver.football.panel/resources/lib/common_addon.py

```diff
@@ -59,3 +62,16 @@ def removeNonAscii(s):
 
 def translate(text):
 	return addon.getLocalizedString(text).encode('utf-8')
+
+def filewrite(_file,contents):
+    f = open(_file, "w")
```

Comment:
would be nice to use "with" statement for file operations to ensure that they get closed properly in case of error.